lib/assert.js

```diff
@@ -79,6 +79,10 @@ function getMessage(self) {
 // display purposes.
 
 function fail(actual, expected, message, operator, stackStartFunction) {
+  if (arguments.length === 1)
+    message = actual;
+  if (arguments.length === 2)
+    operator = '!=';
```

Comment:
You could just write `message = actual, operator = '!=', stackStartFunction = undefined` as part of the function arguments, right?